channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -263,7 +259,6 @@ export function PageViewEnvelopeCreator(logger: IDiagnosticLogger, telemetryItem
     EnvelopeCreatorInit(logger, telemetryItem);
 
     // Since duration is not part of the domain properties in Common Schema, extract it from part C
-    let strDuration = "duration";
```

Comment:
Moved to InternalConstants.ts, so all usages in the project will use this value.